src/snowflake/snowpark/session.py

```diff
@@ -784,6 +810,24 @@ def large_query_breakdown_enabled(self, value: bool) -> None:
                 "value for large_query_breakdown_enabled must be True or False!"
             )
 
+    @large_query_breakdown_complexity_bounds.setter
+    def large_query_breakdown_complexity_bounds(self, value: Tuple[int, int]) -> None:
```

Comment:
might be easier to take a lower and upper bound there, then you can reconstruct the tuple internally, and we can make lower or upper optional also

Reply:
@sfc-gh-yzou this is a `@setter` method. I'm afraid we have to abide by this format.